hack/jenkins/e2e.sh

```diff
@@ -245,6 +240,8 @@ if [[ ${JOB_NAME} =~ ^kubernetes-.*-gce ]]; then
 elif [[ ${JOB_NAME} =~ ^kubernetes-.*-gke ]]; then
   KUBERNETES_PROVIDER="gke"
   : ${E2E_ZONE:="us-central1-f"}
+  : ${CLOUDSDK_BUCKET:="gs://cloud-sdk-build/testing/staging"}
```

Comment:
Worth adding an explanatory comment here that gke tests at head run against the test endpoint using a ci gcloud build?